.github/workflows/ci.linux.arm.yml

```diff
@@ -63,8 +63,10 @@ jobs:
           source /opt/rh/gcc-toolset-9/enable
           cmake -B build \
             -D CMAKE_BUILD_TYPE=Debug \
+            -D PHOTON_BUILD_DEPENDENCIES=ON \
```

Comment:
Since we have been using custom image, at least keep one CI to verify `PHOTON_BUILD_DEPENDENCIES`.